source/docs/guide/src/ide_support.md

```diff
@@ -0,0 +1,49 @@
+# Using Verus with rust-analyzer
+
+
+After following the below steps, when you save a file, rust-analyzer will run Verus for that file and display generated error messages. By default, rust-analyzer uses `cargo check` to get diagnostics, but we intercept that and make the rust-analyzer use Verus. 
+
+### 1. Compile custom rust-analyzer
+
+1. Clone the repository: `git clone https://github.com/channy412/rust-analyzer/tree/connect-verus`
+2. `cd rust-analyzer`
+3. Compile the rust-analyzer binary: `cargo xtask dist`
+4. Unzip the generated file (e.g. `gunzip ./dist/rust-analyzer-x86_64-apple-darwin.gz`)
+5. Make it executable (e.g. `chmod +x ./dist/rust-analyzer-x86_64-apple-darwin`)
```

Comment:
This seems specific to macOS. Do you know if this would work on other platforms? If not, it's probably best to mention that this is currently only tested on macOS.